gnrpy/gnr/web/gnrtask.py

```diff
@@ -27,12 +27,33 @@
 from datetime import datetime
 from time import sleep
 from random import randrange
-
+from gnr.core.gnrconfig import getGnrConfig
 from gnr.app.gnrapp import GnrApp
 from gnr.core.gnrbag import Bag
 from gnr.web.gnrwsgisite import GnrWsgiSite
 from gnr.web import logger
 
+
+def determine_task_manager_to_use():
+    """
+    Load the task async implementation configuration from
+    environment.xml. If it fails, for example by importing the code
+    in a non-established environment, default back to old implementation anyway.
+    """
+    try:
+        _c = getGnrConfig()
+        _env = _c['gnr.environment_xml']
+        task_configuration = _env.getAttr('tasks')
```

Comment:
```suggestion
        task_configuration = _env['environment'].getAttr('tasks')
```

**Critical Bug #1: Configuration not loaded correctly**

Issue: The `determine_task_manager_to_use`() function fails to read the async_impl configuration from environment.xml, preventing the new async task system from being activated.

Root cause: Incorrect XML path - the function tries to read tasks attributes directly from the root environment bag instead of from the environment node.

Testing: After this fix, **USE_ASYNC_TASKS** correctly returns True when `<tasks async_impl="true" /> `is set in environment.xml.